src/Components/Highlight/Highlight.js

```diff
@@ -1,25 +1,62 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
 import './Highlight.css';
+import apiCalls from '../../apiCalls';
 import images from '../../data/imagesData';
+import Loading from '../Loading/Loading';
 
 const Highlight = () => {
-  const getRandomIndex = (array) => {
-    return Math.floor(Math.random() * array.length);
-  }
+  const [bird, setBird] = useState(null);
+  const [highlightImg, setHighlightImg] = useState('')
 
-  const birdID = Object.keys(images)
-  const randomBird = birdID[getRandomIndex(birdID)]
 
-  const birdImage = () => {
-    return images[randomBird]
-  }
+  useEffect(() => {
```

Comment:
Moving everything into the `useEffect` created a clearer understanding of what data I'm generating and what data I'm trying to display.